src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset.scss";
+
+@import "./utils/normalize.scss";
+
+@import "./utils/variableses.scss";
+
+@import "./utils/function.scss";
+
+@import "./blocks/calendar.scss";
```

Comment:
Why do you need these spaces between imports?

Reply:
fixed